editor.planx.uk/src/routes/authenticated.tsx

```diff
@@ -26,9 +27,22 @@ export type TeamSummary =
   { settings: Pick<TeamSettings, "isTrial"> } &
   { theme: Pick<TeamTheme, "primaryColour" | "logo"> }
 
+interface Context {
+  user: User
+}
+
 const editorRoutes = compose(
   withView(authenticatedView),
 
+  /**
+   * Ensure userStore is initialised
+   * Required for route guards to avoid race conditions
+   */
+  withContext(async (): Promise<Context> => {
+    const user = await useStore.getState().initUserStore();
+    return { user }
+  }),
```

Comment:
:raised_hands: simple solution, very effective !

Reply:
I know....! 🤦‍♂️ Spent a long time to get these few lines!